hubmap/portal/views.py

```diff
@@ -0,0 +1,59 @@
+import string
+
+from django.shortcuts import render
+
+entities_list = [{"entity_type": "sample", "hubmap_id": 1},
```

Comment:
What are the numbers 1 through 6? These look like database primary keys, but they don't seem to correspond to any database model. Is this test data?

Reply:
As per #173 summary, we would not be using models for now. I think Chuck mentioned that there will be no models and mentioned that Data will be pulled from the API, which is not under Django’s ORM.

Reply:
PR #181 implements a API client class.